web-gateway/app/controllers/ItemController.java

```diff
@@ -33,23 +35,30 @@
     private final BiddingService bidService;
 
     private final Boolean showInlineInstruction;
+    private HttpExecutionContext ec;
```

Comment:
Yeah, I did. This was added in Play 2.5 but it didn't create any deprecation warning.
There are now separate Execution contexts and you must use the HTTP one for http rendering.